lib/serverengine/process_manager.rb

```diff
@@ -18,12 +18,17 @@
 module ServerEngine
 
   require 'fcntl'
+  if ServerEngine.windows?
+    require 'win32/pipe'
+    include Win32
```

Comment:
let's avoid including a namespace to make using external libraries explicit. You can still use `Win32::Pipe::Server`.